sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/internal/SensorContextTester.java

```diff
@@ -122,6 +124,11 @@ public Settings settings() {
     return settings;
   }
 
+  @Override
+  public Configuration config() {
+    return new ConfigurationBridge(settings);
+  }
+
   public SensorContextTester setSettings(Settings settings) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Settings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)